src/System.Linq.Parallel/tests/WithCancellationTests.cs

```diff
@@ -14,24 +14,23 @@ namespace System.Linq.Parallel.Tests
     // the tests here are only regarding basic API correctness and sanity checking.
     public static class WithCancellationTests
     {
-        [Fact]
-        public static void MultiplesWithCancellationIsIllegal()
+        [Theory]
+        [MemberData(nameof(Sources.Ranges), new[] { 16 }, MemberType = typeof(Sources))]
+        public static void WithCancellation_Multiple_NotCancelable(Labeled<ParallelQuery<int>> labeled, int count)
         {
-            InvalidOperationException caughtException = null;
-            try
-            {
-                CancellationTokenSource cs = new CancellationTokenSource();
-                CancellationToken ct = cs.Token;
-                var query = Enumerable.Range(1, 10).AsParallel().WithDegreeOfParallelism(2).WithDegreeOfParallelism(2);
```

Comment:
[Yes](https://github.com/dotnet/corefx/blob/master/src/System.Linq.Parallel/tests/DegreeOfParallelismTests.cs#L149-L152) (Part of the earlier work)